storage/allocator.go

```diff
@@ -251,9 +251,9 @@ func (a *Allocator) AllocateTarget(required roachpb.Attributes, existing []roach
 			return target, nil
 		}
 		if len(attrs) == 0 {
-			return nil, util.Errorf("unable to allocate a target store; no candidates available")
+			return nil, util.Errorf("no suitable replication target store found, are you running enough nodes?")
```

Comment:
The StorePool should have enough information to recreate this; it has the details of every store, it can compute the number of nodes from that if desired.